src/FAQ.mdx

```diff
@@ -0,0 +1,46 @@
+---
+name: FAQ
+---
+
+# Frequently Asked Questions
+
+The Blue Design System Mission Statement: Establish a system that provides a single source of truth to ease the creation of high quality user experiences.
+
+## What is Blue Design System?
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+## Who is the design system for?
+
+The Blue Design System delivers a single source of truth for designers and developers to ease the creation of high quality user experiences.
+
+The Blue Design System is built with the following personas in mind:
+
+- The UX Designer, who wants to provide user experiences that are high quality, accessible, and consistent
+- The UI Developer, who wants their code to be clean and consistent and conform to accessibility standards
+- The Product Owner, who wants their product teams to focus on the hard stuff not the easy stuff so product development moves faster and delivers consistent and reliable product experiences every time
+- The Product Stakeholders, who want to be assured of the accessibility, consistency, and quality of the user experience
+
+## How do I get the latest Blue Design System library?
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```

Comment:
- [ ] See our [Installation](/#/installation) page for instructions on getting the latest version of the Blue Design System installed.